### DISCUSSION TITLE
Files Connector AzureBlob: Operation returned an invalid status 'The specifed resource name contains invalid characters.'\nErrorCode:InvalidResourceName

### DISCUSSION THREAD

**Discussion by Ruitjes** [airbytehq/airbyte#35789] [Connector Questions]
Hello all,

I was busy with creating a connections between Files (stored in Azure Blob Storage), and MSSQL. However, I'm getting the following error: [check_connection_source-failure-9c089cc3-0272-4700-bc36-6e0e4439cd54.txt](https://github.com/airbytehq/airbyte/files/14480981/check_connection_source-failure-9c089cc3-0272-4700-bc36-6e0e4439cd54.txt)

As seen in the logs, the URL is being converted to `azure://` but i'm no sure if the right behaviour is anyone able to check if this a mistake on my side or a bug Airbyte. I also saw the same issue here: https://github.com/airbytehq/airbyte/issues/24955

![image](https://github.com/airbytehq/airbyte/assets/26769441/2a602ee5-61ab-4be0-ac60-c53ba88997a0)

Thanks in advance!

**Comment by akozichev** [comment]
for azure blob storage URL should only contain container name and file name, see example https://metaops.solutions/blog/processing-hl7-with-airbyte-and-dbt

**Reply by Ruitjes** [reply]
Thanks @akozichev, was reading the documentation again and I know see it's also documented.
<img width="841" alt="image" src="https://github.com/airbytehq/airbyte/assets/26769441/ec08f228-7c93-43f9-a063-b81915ab7fa4">
